### PR TITLE
Initialize google analytics on Academix

### LINKED ISSUE #55: No method to keep in track the users details
**Is your feature request related to a problem?**
We should use some kinda analytics service to keep track of user data who's using Academix. 

**Describe the solution you'd like**
Since already we are using google analytics in SEF website, We can use the same method for Academix too.

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #55 

## Goals

Implement a way to track user's activities

## Approach

Initialize Google analytics
Created a util for track views

Screencast from 2020-07-27 13_46_46

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Learning

https://medium.com/google-cloud/tracking-site-visits-on-react-app-hosted-in-google-cloud-using-google-analytics-f49c2411d398

### CODE REVIEW

File: app/src/components/App/index.tsx [hunk 1]
```diff
@@ -15,6 +16,9 @@ import {
   MenuOutlined,
 } from '@ant-design/icons';
 
+const trackingId = 'UA-167873271-2';
+ReactGA.initialize(trackingId);
```
Comment (jayasanka-sack) [inline app/src/components/App/index.tsx:20]:
```suggestion
ReactGA.initialize('UA-167873271-2');
```

File: app/src/services/util/track.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import ReactGA from 'react-ga';
+
+export const trackPage = () => {
```
Comment (jayasanka-sack) [inline app/src/services/util/track.js:3]: Shall we refactor the function name?

something like `trackPageWithGoogleAnalytics`

Comment (jayasanka-sack) [reply app/src/services/util/track.js:3]: Lets change the file name to `googleAnalytics.js`

File: app/src/components/App/index.tsx (hunk 1 above)
Comment (jayasanka-sack) [reply app/src/components/App/index.tsx:20]: Also, add a comment before this line.

**Comment by Piumal1999** [comment]
> WoW! Great job @Piumal1999 !!
> 
> I've added some comments. Apart from that, is this going to update when we run this on locally? If yes, is there a way to prevent it?

Yeah. I added a filter from the google analytics dashboard. Now it'll only update using data taken from sefglobal.org hostname.

File: app/src/components/App/index.tsx (hunk 1 above)
Comment (Piumal1999) [reply app/src/components/App/index.tsx]: Done

File: app/src/services/util/track.js (hunk 2 above)
Comment (Piumal1999) [reply app/src/services/util/track.js]: Okay. Updated

**Pull request merged**